_practicalities/intro.md

```diff
@@ -1,16 +1,22 @@
 ---
+bg: '#4da45e'
+color: white
+title: practicalities
+fa-icon: pencil
+id: bar
 ---
 
-Add note here to explain that this does _not_ prevent _nor_ discourage library
-author to release 2 version of their software one Python 3 only and the other
-python 2.
+We do not discourage authors to release software on Python 2. While this guide
+is mostly written with the assumption that software are going to stop Python 2
+support, it does perfectly apply to a package that wish to not support Python 3,
```

Comment:
I'm not quite sure what you're saying here. How would this apply to a package that doesn't wish to support Python 3?